public/app/plugins/panel/heatmap/HeatmapHoverView.tsx

```diff
@@ -287,3 +236,68 @@ const HeatmapHoverCell = ({ data, hover, showHistogram, scopedVars, replaceVars
     </>
   );
 };
+
+export function renderHistogram(
```

Comment:
this was all extracted for reuse with only non-effectual changes, except the new bar colors (below)